### PR TITLE
Updated feature/improve test suite

### PR DESCRIPTION
**Pull request by jayair**
@carlcheel Created a new branch here by mistake as I was working on https://github.com/AnomalyInnovations/serverless-bundle/pull/110

This update cleans up the `tests/scripts` test we were manually running. So the `test` script in the `package.json` is a lot cleaner now.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1 +1 @@
-tests/*
+tests
```
Comment (carlcheel) [inline .eslintignore:1]: Do we any to remove lint from tests. I removed this from the original and it highlighted a few issues. With `jest: true` in the eslint config, it should work fine or is there another reason why you've disabled it?

**Comment by carlcheel** [comment]
GitHub went down earlier and was getting 500 errors, hence all the extra approvals.

File: .eslintignore (hunk 1 above)
Comment (jayair) [reply .eslintignore:1]: Yeah I found that Husky complains with listing errors when you try to commit. Does it not do that for you?

Comment (carlcheel) [reply .eslintignore:1]: Just gave it another try, works fine for me. Runs `eslint --fix` and passes... wonder if it's not picking up the eslint config.

husky-serverless-bundle

Comment (jayair) [reply .eslintignore:1]: Ah sorry for the confusion. Yeah previously I didn't want linting for the tests. But if you've fixed all the tests. Then let's stick with it.

**Comment by jayair** [comment]
Thank you @carlcheel!

**Pull request merged**